### DISCUSSION TITLE
报这个错请问怎么解决

### DISCUSSION THREAD

**Discussion by zh758631301** [deepjavalibrary/djl#1712] [Q&A]
![image](https://user-images.githubusercontent.com/49860741/173530479-723ae44e-910d-457d-a497-794ba74f55fc.png)

aarch64架构的cpu，在集成DJL使用api0.17.0中的ModelZoo.loadModel(recognition.detectCriteria())方法出现如下问题，具体在CudaUtils的类中loadLibrary()方法中执行Native.load("cudart", CudaLibrary.class)时出错，Native使用的是jna5.8.0中的类

**Comment by frankfliu** [comment]
It seems you didn't include JNA dependency in your project.

Can you check if JNA is in your classpath.
